Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -146,11 +150,18 @@ cannot be guaranteed to be immutable.
 The listeners
 -------------
 
-Extensions and PHP packages can add listeners that are registered via YAML. They
+Extensions and PHP packages can add listeners that are registered via YAML (or starting
+with TYPO3 v13 also natively in :ref:`PHP using attributes <EventDispatcherImplementation>`). They
```

Comment:
Here you have to take care when backporting. I suggest to remove the mentioning of the attributes for this PR and add it in a separate PR (then only for main).

Reply:
IMO when people are reading the documentation of this in a v11/v12 environment it would be good for adoption and getting "people hooked" ;) to spread the word about the cool new attributes? It does mention v13 specifically, so I'd really like to keep it not only for preventing the hassle of an additional PR and drifting changes in this section?

Reply:
No, this is not how the docs work, this is not marketing/sales where we want to sell something.
When people have to look up the docs because of a problem or feature they want to implement, they can't go to the customer and say: Hey, I want to use the cool attributes, let's upgrade spontaneously to v13.0.
And in older versions they can't "adopt" the attribute.
Besides of that: the attribute is just another way of registering an event listener. Nothing else.